app.js

```diff
@@ -3,15 +3,14 @@ const {
   getCategories,
   getReviewsCommentCount,
   getReviewsWithReviewId,
+  getReviewComments,
 } = require("./controllers/games-controller");
 const app = express();
 
 app.get("/api/categories", getCategories);
 app.get("/api/reviews", getReviewsCommentCount);
 app.get("/api/reviews/:review_id", getReviewsWithReviewId);
-app.use((err, req, res, next) => {
```

Comment:
Has this been removed? It is still required but after the line with the   "/api/reviews/:review_id/comments" endpoint